service/frontend/api/producer_manager.go

```diff
@@ -70,7 +70,7 @@ func NewProducerManager(
 			InitialCapacity: 5,
 			MaxCount:        100,
 			Pin:             true,
-			MetricsScope:    metricsClient.Scope(metrics.Frontend),
+			MetricsScope:    metricsClient.Scope(metrics.PersistenceGetShardScope), // was metrics.Frontend, using incorrect int
```

Comment:
clear incorrect value, was a ServiceIdx when it needed a ScopeIdx.

I've replaced it with the metric with the same operation (`1`) so metrics are unchanged, but this is almost certainly wrong.  how do we use this?

Reply:
given it's the async APIs, I imagine we've just not used it heavily yet